src/extension/devtools/devtools.ts

```diff
@@ -122,29 +225,20 @@ devtools.listen<string>(CREATE_DEVTOOLS_PANEL, async ({ payload }) => {
       `background:tab-${inspectedTabId}:client`
     );
 
-    // Listen for tab reload from background
-    removeReloadListener = devtools.listen(RELOADING_TAB, () => {
-      handleReload();
-      clearRequestInterval();
-
-      const removeListener = devtools.listen(RELOAD_TAB_COMPLETE, () => {
-        clearRequestInterval = startRequestInterval();
-        handleReloadComplete();
-        removeListener();
-      });
-    });
+    panelHidden = false;
   });
 
   panel.onHidden.addListener(() => {
-    isPanelCreated = false;
```

Comment:
Hiding the panel happens when the user switches tabs in the devtools, not just closing them altogether. The notion that the panel was no longer created here before was incorrect.